src/test/crimson/seastore/CMakeLists.txt

```diff
@@ -6,7 +6,9 @@ add_ceph_unittest(unittest_transaction_manager)
 target_link_libraries(
   unittest_transaction_manager
   ${CMAKE_DL_LIBS}
-  crimson-seastore)
+  crimson-seastore
+  crimson-os
+  ceph-common)
```

Comment:
why would you want to link `unittest_transaction_manager` against `ceph-common`? please note, we want to avoid linking against `ceph-common` in crimson. as the implementation of `crimson-common` is different than `ceph-common`. the former is lockless.

why do you want to link against crimson-os ?

Reply:
> why would you want to link `unittest_transaction_manager` against `ceph-common`? please note, we want to avoid linking against `ceph-common` in crimson. as the implementation of `crimson-common` is different than `ceph-common`. the former is lockless.

will try crimson-common instead
> 
> why do you want to link against crimson-os ?

because of:
```
/usr/bin/ld: ../../../../../lib/libcrimson-cyanstore.so: undefined reference to `ceph::os::Transaction::dump(ceph::Formatter*)'
```